include/dpp/discordvoiceclient.h

```diff
@@ -50,14 +50,18 @@
 #include <functional>
 #include <chrono>
 
-
-
 struct OpusDecoder;
 struct OpusEncoder;
 struct OpusRepacketizer;
 
 namespace dpp {
 
+inline constexpr uint16_t AUDIO_TRACK_MARKER = 0xFFFF;
+
+inline constexpr int AUDIO_OVERLAP_SLEEP_SAMPLES = 30;
```

Comment:
This doesn't follow the [coding style standards](https://dpp.dev/coding-standards.html#:~:text=Class%20names%2C%20function%20names%20and%20method%20names). I am aware that it was previously all caps before which does confuse me :P

Reply:
I will revert those to macro as those are out of scope of this PR

Reply:
Sounds good!